client/src/site-search/search-results.scss

```diff
@@ -8,11 +8,23 @@
     font-style: italic;
     font-size: 0.9rem;
   }
+  span.summary {
+    display: block;
+    font-size: 0.9rem;
+  }
   span.nerd-data {
     margin-left: 20px;
     font-size: 80%;
   }
 
+  // Used to indicate that a search result link is in a different locale
+  .locale-indicator {
+    font-size: 80%;
+    background-color: bisque;
+    padding: 3px;
```

Comment:
Generally, `px` values scale badly with user zooming in (e.g. via <kbd>Ctrl</kbd>+<kbd>+</kbd>)

Reply:
Ignore my hacky CSS. I expect @schalkneethling to overwrite most of the SCSS the proper way.

Reply:
It could be solved by not using CSS at all in Yari